### PR TITLE
Update the README based off local setup experience.

### PR DESCRIPTION
**Pull request by sarahboyce**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,6 +184,8 @@ Don't forget to give the project a star! Thanks again!
 4. Push to the Branch (`git push origin feature/AmazingFeature`)
 5. Open a Pull Request
 
+See the [open issues](https://github.com/dawnwages/wagtail-indymeet/issues) for a full list of proposed features (and known issues).
+
```
Comment (tim-schilling) [inline README.md:188]: At the top there is a "Request Feature" and "Report Bug" which should point to the issues area (#186), I'm not sure this is needed?

File: .env.template.local [hunk 2]
```diff
@@ -0,0 +1,5 @@
+DATABASE='djangonaut_space'
```
Comment (tim-schilling) [inline .env.template.local:1]:
```suggestion
DATABASE='djangonaut-space'
```

File: README.md (hunk 1 above)
Comment (sarahboyce) [reply README.md:188]: Ah OK, none of the three links at the top point to anything. Shall I delete the top ones or this? I don't mind 🤔

File: .env.template.local [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+SECRET_KEY=<SECRETKEY>
+ENVIRONMENT='local'
```
Comment (tim-schilling) [inline .env.template.local:3]:
```suggestion
```
`ENVIRONMENT` is only needed for production.

**Review by tim-schilling — CHANGES_REQUESTED** [review]
The two changes to the template file I think you should make, but the README one is up to you.

File: README.md (hunk 1 above)
Comment (tim-schilling) [reply README.md:188]: #186 will populate those links.

**Review by tim-schilling — APPROVED** [review]
:rocket:

**Pull request merged**